File: 1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    // Initialize variables
    float input;
    int cents;
    float quarter = 25;
    float dime = 10;
    float nickel = 5;
    float penny = 1;
    int counter = 0;
    // Prompt user for change input
    do
    {
        // Make sure input is more than zero
        input = get_float("Change owed: ");
        cents = round(input * 100);
    }
    while (input < 0);
    
    // Calcaulate the number of coins
    while (cents >= quarter)
    {
        cents = cents - quarter;
        counter++;
    }
    while (cents >= dime)
    {
        cents = cents - dime;
        counter++;
    }
    while (cents >= nickel) 
    {
        cents = cents - nickel;
        counter++;
    }
    while (cents >= penny)
    {
        cents = cents - penny;
        counter++;
    }
    // Print the output
    printf("%i\n", counter);
}

File: 6/cash/cash.py
#!/usr/bin/env python3
# Cash

from cs50 import get_float


def get_change():
    # get the round value of input
    while True:
        change = get_float("Change owed: ")
        cents = round(change * 100)
        if change > 0:
            return cents


def main():
    cents = get_change()
    quarter = 25
    dime = 10
    nickel = 5
    penny = 1
    counter = 0
    # start looping and counting
    while cents >= quarter:
        cents = cents - quarter
        counter = counter + 1
    while cents >= dime:
        cents = cents - dime
        counter = counter + 1
    while cents >= nickel:
        cents = cents - nickel
        counter = counter + 1
    while cents >= penny:
        cents = cents - penny
        counter = counter + 1
    print(counter)


if __name__ == '__main__':
    main()

